### ISSUE TITLE
Proper git tag for version please?

### ISSUE THREAD

**Issue by cicku** [JohnCremona/eclib#9]
Hi,

May I ask for a change of tags from eclib-$VER to $VER or v$VER? It will help us tracking the update. Currently we only track 20150323 because it matches the syntax of "commonly used tags".

Thanks.

**Comment by cicku** [comment]
How we track:

https://release-monitoring.org/project/7361/

**Comment by JohnCremona** [comment]
Who are you?  Who is "we"?  I get many requests asking me to do things differently, and I try to be amenable, but there are a lot of conventions out there and I don't even know if they are compatible.
Since I started adding versioning (rather recently) I have made the git tags agree with the tarball name - except when I got that wrong, which seems to be the only version you (whoever you are) "track" (whatever that means).  But I see no harm in having the tag v20150826 to match the tarball eclib-20150826.tar.bz2.
In any case I will make a new version today since I realised that in yesterday's bug-fix release I forgot to increment the bit of the library name.
